daily-challenges/2024-11/07/ththao27.spec.ts

```
@@ -0,0 +1,70 @@
+        }
+        else {
+            result += char;
+          }
+    }
+    return result;
+}
+
+//1. **Thêm chức năng giải mã:**
+
+function caesarDecipher (massages: string[], shift: number): string[] {
+    shift = shift % 26;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)